pkgs/by-name/vs/vscode-runner/package.nix

```diff
@@ -0,0 +1,57 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildDartApplication,
+  sqlite,
+  kdePackages,
+}:
+
+let
+  pname = "vscode-runner";
+  version = "1.6.1";
+
+  src = fetchFromGitHub {
+    owner = "Merrit";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-mDhwydAFlDcpbpmh+I2zjjuC+/5hmygFkpHSZGEpuLs=";
+  };
+in
+buildDartApplication {
+  inherit pname version src;
+
+  vendorHash = "sha256-jS4jH00uxZIX81sZQIi+s42ofmXpD4/tPMRV2heaM2U=";
+
+  pubspecLock = lib.importJSON ./pubspec.lock.json;
+
+  dartEntryPoints = {
+    "bin/vscode_runner" = "bin/vscode_runner.dart";
+  };
+
+  preFixup = ''
+    wrapProgram $out/bin/vscode_runner \
+      --prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ sqlite ]}
```

Comment:
Maybe we could use patchelf instead

Reply:
i don't know how to use it

do you have any doc or example with a similar case to help me ?

Reply:
Something like https://github.com/NixOS/nixpkgs/blob/7c92ddf469d6d6c236afdcb801a962e9ce03884c/pkgs/applications/misc/holochain-launcher/default.nix#L46 probably

Reply:
it seems that it is not needed anymore, it should be legacy from first tests